### PR TITLE
Various improvements to benchmark mode, plus a glyph stress benchmark.

### PR DESCRIPTION
**Pull request by glennw**
* Track the CPU backend time in benchmark mode.
* Add a glyph rendering stress test.
* Make subpixel AA default text mode in wrench.
* Rebuild display lists every frame in benchmark mode.

---
This change is [Reviewable](https://reviewable.io/reviews/servo/webrender/1514)

### CODE REVIEW

**Comment by glennw** [comment]
Rebased.

File: wrench/src/args.yaml [hunk 1]
```diff
@@ -24,10 +24,10 @@ args:
       long: save
       help: 'Save frames, one of: yaml, json, or binary'
       takes_value: true
-  - subpixel_aa:
+  - disable_subpixel_aa:
```
Comment (kvark) [inline wrench/src/args.yaml]: I think it should be `no_subpixel_aa` for consistency with other options

File: wrench/src/main.rs [hunk 2]
```diff
@@ -320,7 +320,7 @@ fn main() {
                                  save_type,
                                  size,
                                  args.is_present("rebuild"),
-                                 args.is_present("subpixel_aa"),
+                                 !args.is_present("disable_subpixel_aa"),
```
Comment (kvark) [inline wrench/src/main.rs]: similarly, we might want to move the inversion `!` later down the pipe in order to be consistent with the other options

File: wrench/src/perf.rs [hunk 3]
```diff
@@ -188,30 +190,28 @@ impl<'a> PerfHarness<'a> {
...
-                                         .sum();
+        let composite_time_ns = extract_sample(&mut cpu_frame_profiles, |a| a.composite_time_ns);
+        let paint_time_ns = extract_sample(&mut gpu_frame_profiles, |a| a.paint_time_ns);
+        let backend_time_ns = extract_sample(&mut cpu_frame_profiles, |a| a.backend_time_ns);
 
         TestProfile {
             name: filename.to_str().unwrap().to_string(),
-            composite_time_ns: composite_time / cpu_samples.len() as u64,
-            paint_time_ns: paint_time / gpu_samples.len() as u64,
+            composite_time_ns: composite_time_ns,
```
Comment (kvark) [inline wrench/src/perf.rs]: nit: looks like you haven't gotten used to the new shiny Rust 1.16 features yet :)

File: wrench/src/perf.rs [hunk 4]
```diff
@@ -188,30 +190,28 @@ impl<'a> PerfHarness<'a> {
...
+            paint_time_ns: paint_time_ns,
+            backend_time_ns: backend_time_ns,
             draw_calls,
         }
     }
 }
 
+fn extract_sample<F, T>(profiles: &mut [T], f: F) -> u64 where F: Fn(&T) -> u64 {
```
Comment (kvark) [inline wrench/src/perf.rs]: I like this one 👍

**Comment by glennw** [comment]
@kvark Thanks, fixed those nits.

**Pull request merged**